core/cat/mad_hatter/plugin.py

```diff
@@ -241,6 +241,16 @@ def _load_manifest(self):
         meta["thumb"] = json_file_data.get("thumb", "")
         meta["version"] = json_file_data.get("version", "0.0.1")
 
+        # Core compatibility
+        compatibility = json_file_data.get(
+            "compatibility",
+            {
```

Comment:
Please keep the structure flat for the moment, maybe with a more explicit naming, i.e.
`version`, ..., `min_cat_version`, `max_cat_version`